pkg/controller/stack/stack_controller.go

```diff
@@ -1050,6 +1050,7 @@ func (sess *reconcileStackSession) lookupPulumiAccessToken(ctx context.Context)
 // the directory is expected to be removed after processing, this won't cause collisions; but, we
 // check anyway, treating the existence of the build directory as a crude lock.
 func makeWorkingDir(s *pulumiv1.Stack) (_path string, _err error) {
+	//	d, _ := os.MkdirTemp("", "pulumi-build")
```

Comment:
I can take this out; I left it here because it was convenient for checking whether the test passed when the build directory was _not_ stable (and the go build cache thereby got new files in it).

Reply:
Agreed - this should probably be omitted now.